server/dbutils.go

```diff
@@ -0,0 +1,54 @@
+package server
+
+import (
+	"context"
+
+	"github.com/joshprzybyszewski/cribbage/model"
+	"github.com/joshprzybyszewski/cribbage/server/persistence"
+	"github.com/joshprzybyszewski/cribbage/server/play"
+)
+
+func handleAction(db persistence.DB, action model.PlayerAction) error {
```

Comment:
🌵 let's pass in the `_ context.Context` as the first param here from the caller. Apparently it's "best practice" and we may want it in the future.

Reply:
Hmm I'm happy to do that - but doesn't this kinda conflict with the Go philosophy of "don't add stuff that's unused"? What exactly is the best practice this would follow?